docs/USERGUIDE.md

```diff
@@ -71,6 +74,29 @@ NAME       STATE
 myubuntu   Available
 ```
 
+### 2. Import image from Local source
+
+```shell
+# Deploy local upload proxy CR.
+$ kubectl apply -f deploy/crds/hypercloud.tmaxanc.com_v1alpha1_localuploadproxy_cr.yaml
+
+# Wait until local upload proxy is ready to use and the command is not null
+$ kubectl get lup
+NAME       READYTOUSE   COMMAND
+myubuntu   True         < /filePath/file.name kubectl exec -i myubuntu-upload-pod -n default -- dd of=/dev/block-volume
+
+# Change the `/filePath/file.name` in the command string and execute the command to upload the raw type image
```

Comment:
Adding another line of explanation would be nice to specify which type of image file is supported. (Only raw type? or qemu cow image as well? )